controllers/cinder_controller.go

```diff
@@ -878,6 +889,25 @@ func (r *CinderReconciler) apiDeploymentCreateOrUpdate(ctx context.Context, inst
 }
 
 func (r *CinderReconciler) schedulerDeploymentCreateOrUpdate(ctx context.Context, instance *cinderv1beta1.Cinder) (*cinderv1beta1.CinderScheduler, controllerutil.OperationResult, error) {
+
+	svcTemplate := instance.Spec.CinderScheduler
+	cinderSchedulerSpec := cinderv1beta1.CinderSchedulerSpec{
+		ContainerImage:             svcTemplate.ContainerImage,
+		Replicas:                   svcTemplate.Replicas,
+		Debug:                      svcTemplate.Debug,
+		CustomServiceConfig:        svcTemplate.CustomServiceConfig,
+		CustomServiceConfigSecrets: svcTemplate.CustomServiceConfigSecrets,
+		DefaultConfigOverwrite:     svcTemplate.DefaultConfigOverwrite,
+		NetworkAttachments:         svcTemplate.NetworkAttachments,
+		ServiceUser:                instance.Spec.ServiceUser,
+		DatabaseHostname:           instance.Status.DatabaseHostname,
+		DatabaseUser:               instance.Spec.DatabaseUser,
+		Secret:                     instance.Spec.Secret,
+		ExtraMounts:                instance.Spec.ExtraMounts,
+		TransportURLSecret:         instance.Status.TransportURLSecret,
+		PasswordSelectors:          instance.Spec.PasswordSelectors,
```

Comment:
We need to ExternalEndpoints here as well or we need to remove it from the common template if that is not a common field

Reply:
(see comment above)